src/rime/algo/syllabifier.h

```diff
@@ -15,15 +15,22 @@
 namespace rime {
 
 class Prism;
+class Corrector;
 
 using SyllableId = int32_t;
 
-using SpellingMap = map<SyllableId, SpellingProperties>;
+struct EdgeProperties : SpellingProperties {
```

Comment:
I don't see a strong need to override, maybe just add the field in `SpellingProperties`.

Reply:
Intuitively `SpellingProperties` should be consistent with `SpellingDescriptor` saved in prism?